### PR TITLE
[Fix] Prepare for merging `refactor-detr`

### PR DESCRIPTION
**Pull request by Li-Qingyun**
We (me, @jshilong, @LYMDLUT, and @KeiChiTse) have refactored DETR-like models, to enhance the usability and readability of our codebase. This PR fix the branch `refactor-detr` to prepare for merging.

- [x] update links in readme of Mask2Former (adding the latest checkpoint of better PQ)
- [x] fix lacking of copyright announcement
- [x] fix unit test file of Mask?Former pixel decoder plugins
- [ ] click and test the link after uploading the new model and log

### CODE REVIEW

**Comment by Li-Qingyun** [comment]
Invite @jshilong to review ~ 3g u

**Review by jshilong — COMMENTED** [review]
Is all copyright line added for new files?

**Comment by Li-Qingyun** [comment]
> Is all copyright line added for new files?

yes, I've checked

**Comment by jshilong** [comment]
We need to update the checkpoint after the Chinese new year holiday with the help of @hhaAndroid

**Comment by jshilong** [comment]
Please check if is there any else that needs to do for refactor-detr, we should add them to this pr

**Comment by Li-Qingyun** [comment]
> Please check if is there any else that needs to do for refactor-detr, we should add them to this pr

okkkkk

File: configs/mask2former/README.md [hunk 1]
```diff
@@ -42,16 +42,14 @@ mmdetection
...
+| R-50     | pytorch | ImageNet-1K  | 50e     | 13.9     | -              | 52.0 | 44.5    | 41.8     | [config](./mask2former_r50_8xb2-lsj-50e_coco-panoptic.py)                       | [model](https://download.openmmlab.com/mmdetection/v3.0/mask2former/mask2former_r50_8xb2-lsj-50e_coco-panoptic/mask2former_r50_8xb2-lsj-50e_coco-panoptic_20230118_125535-54df384a.pth) \| [log](https://download.openmmlab.com/mmdetection/v3.0/mask2former/mask2former_r50_8xb2-lsj-50e_coco-panoptic/mask2former_r50_8xb2-lsj-50e_coco-panoptic_20230118_125535.log.json)                                                                                       |
```
Comment (ZwwWayne) [inline configs/mask2former/README.md:45]: The link seems to be still unavailable

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**